scraper.rb

```diff
@@ -18,24 +18,26 @@ def scrape(h)
   klass.new(response: Scraped::Request.new(url: url).response)
 end
 
-def scrape_list(term, url)
-  scrape(url => MembersPage).member_urls.each do |mem_url|
+def member_data(term, url)
+  scrape(url => MembersPage).member_urls.map do |mem_url|
     data = scrape(mem_url => MemberPage).to_h.merge(term: term)
     puts data.reject { |_, v| v.to_s.empty? }.sort_by { |k, _| k }.to_h if ENV['MORPH_DEBUG']
```

Comment:
The normal pattern with a move like this is to move the debugging out of the method too…